lesson-15/index.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles/css/main.css">
+    <title>What do we offer</title>
+</head>
+<body>
+    <div class="wrapper">
+        <header>
+            <div class="header__main">
+                <div class="header__menu">
+                    <a href="#">
+                        <img src="./images/logo_triphouse_blue.svg" alt="logo" />
+                    </a>
+                    <ul>
+                        <li class="header__stays"><a href="#">Stays</a></li>
+                        <li class="header__attractions"><a href="#">Attractions</a></li>
+                        <li><div class="header__night"></div></li>
+                        <li><a href="./index_login.html"><div class="header__account"></div></a></li>
+                    </ul>
+                </div>
+                <div class="header__search">
+                    <p class="header__title">Discover stays<br />
+                        to live, work or just relax</p>
+                    <form class="header__form">
+                        <div class="header__destination">
+                            <p>
+                                <label for="search">Your destination or hotel name</label>
+                            </p>
+                            <input id="search" type="text" name="search" placeholder="New York" />
+                        </div>
+                        <div class="header__date">
+                            <div class="header__in">
+                                <p>
+                                    <label for="in">Check-in - </label>
+                                </p>
+                                <input type="date" id="in" />
+                            </div>
+                            <div class="header__out">
+                                <p>
+                                    <label for="out">Check-out</label>
+                                </p>
+                                <input type="date" id="out" />
+                            </div>
+                        </div>
+                        <div class="header__people">
+                            <input class="header__input_adults" type="text" name="adults" placeholder="2 Adults     — " />
+                            <input class="header__input_children" type="text" name="children" placeholder="0 Children   — " />
+                            <input class="header__input_rooms" type="text" name="rooms" placeholder="1 Room" />
+                        </div>
+                        <button type="submit">Search</button>
+                    </form>
```

Comment:
`</form>` must be after `div.header__people_filter`. But you will have problems with style. You will need to fixed it